javascript/modules/client/README.md

```diff
@@ -1,5 +1,4 @@
-Aragon JS SDK Client
----
+## Aragon JS SDK Client
```

Comment:
Why this change?

This is the markdown document title, if we start using `#` or `##`, then everything else needs to be `###` or more

Reply:
Automatically formatted